tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -61,141 +63,4 @@ spec:
(kubectl get internalrequest \"${internalRequest}\" \\\n    -o jsonpath='{.status.conditions[?(@.type==\"Succeeded\")]}')\n  jq '.reason // \"Unset\"'  <<< \"${conditions}\" | tee \"$(results.requestReason.path)\"\n  jq '.message // \"Unset\"' <<< \"${conditions}\" | tee \"$(results.requestMessage.path)\"\n  jq -r '.indexImageDigests' <<< \"${results}\" |  tee \"$(results.indexImageDigests.path)\"\n\n  # the following ones are used to help debugging in case of failure\n  jq -r '.iibLog' <<< \"${results}\"\n  RC=\"$(jq -r '.exitCode' <<< \"${results}\")\"\n\n  # Summarize what happened for the human user. Although the continuation of the pipeline will depend on all\n  # components having the same flag values, we should display them to help the user to understand which of the\n  # component might be misconfigured.\n  if [ \"$mustPublishIndexImage\" = \"true\" ]; then\n    echo \"Index image will be published.\"\n  elif [ \"$fbc_opt_in\" = \"false\" ]; then\n    echo \"Index image will not be published because fbc_opt_in is set to false in Pyxis.\"\n    echo \"If this is the first time you are releasing, make sure you request fbc_opt_in in Pyxis.\"\n  elif [ \"${staged_index}\" = \"true\" ]; then\n    echo \"Index image will not be published because this is a staging release.\"\n  else\n    echo \"Index image will not be published for an unspecified reason.\"\n  fi\n\n  if [ \"$RC\" -ne 0 ]; then\n    echo \"The fragment failed to be processed, check which fragment and its log link above \" \\\n         \"to understand the reason\"\n    exit \"$RC\"\n  fi\ndone\n\n# the flags of all components should match. A single unmatching flag should stop signing and publishing.\nfbcPipelineFlags=$(jq -r '.genericResult | fromjson' <<< \"${results}\" |jq --slurp  '. | unique')\nif [ \"$(jq -r 'length' <<< \"${fbcPipelineFlags}\")\" == 1 ]; then\n  jq -r '.[].sign_index_image' <<< \"${fbcPipelineFlags}\" | tr -d \"\\n\" \\\n    | tee \"$(results.mustSignIndexImage.path)\"\n  jq -r '.[].publish_index_image' <<< \"${fbcPipelineFlags}\" | tr -d \"\\n\" \\\n    | tee \"$(results.mustPublishIndexImage.path)\"\n  jq -r '.[].fbc_opt_in' <<< \"${fbcPipelineFlags}\" | tr -d \"\\n\" | tee \"$(results.isFbcOptIn.path)\"\nelse\n  echo -en \"false\" | tee \"$(results.mustSignIndexImage.path)\"\n  echo -en \"false\" | tee \"$(results.mustPublishIndexImage.path)\"\n  echo -en \"false\" | tee \"$(results.isFbcOptIn.path)\"\nfi\n\n# if there are multiple components but they are for the same ocp version, only a single index_image needs\n# to be signed/published.\nVERSIONS_COUNT=\"$(jq -r '[ .components[].ocp_version ] | unique | length' \"$RESULTS_FILE\")\"\nif [[ \"$LENGTH\" -gt 1 && \"$VERSIONS_COUNT\" == 1 ]]; then\n  # keep only the most up to date build index_image\n  RESULTS_TEMP_FILE=\"/tmp/results-temp.json\"\n  jq -r  '.components = [ .components | sort_by(.updated) | last ]' \"$RESULTS_FILE\" > \"$RESULTS_TEMP_FILE\"\n  mv \"$RESULTS_TEMP_FILE\" \"$RESULTS_FILE\"\nfi\n"
```

Comment:
Whoa, what's this? :) Now that's a long line...

Reply:
gosh. I know, I run the tests locally. The pre-apply-mock.sh sometimes does that and I did not notice in the last push.

Reply:
Yeah, but I'm not sure how it can happen. I never saw anything like it. If you can reproduce it, please ping me and we can debug further.